manifests/security.pp

```diff
@@ -17,17 +17,91 @@
 # Jenkins security configuration
 #
 class jenkins::security (
-  $security_model = undef,
+  $jenkins_management_email         = '',
+  $jenkins_management_login         = '',
+  $jenkins_management_name          = '',
+  $jenkins_management_password      = '',
+  $jenkins_s2m_acl                  = false,
+  $jenkins_ssh_private_key_contents = '',
+  $jenkins_ssh_public_key_contents  = '',
+  $ldap_access_group                = '',
+  $ldap_group_search_base           = '',
+  $ldap_inhibit_root_dn             = 'no',
+  $ldap_manager                     = '',
+  $ldap_manager_passwd              = '',
+  $ldap_overwrite_permissions       = '',
+  $ldap_root_dn                     = 'dc=company,dc=net',
+  $ldap_uri                         = 'ldap://ldap',
+  $ldap_user_search                 = 'uid={0}',
+  $ldap_user_search_base            = '',
+  $security_model                   = undef,
 ){
   validate_string($security_model)
 
   include ::jenkins::cli_helper
 
+  $libdir = $::jenkins::libdir
+
+  file { "${libdir}/.ssh/id_rsa" :
```

Comment:
The management of some Jenkins public/private SSH keys seems unrelated to this LDAP configuration change, what's it here for?